ppr-ui/src/views/newMhrRegistration/HomeOwners.vue

```diff
@@ -129,7 +129,7 @@
         <v-col cols="9" class="mb-n6 pa-0"></v-col> <!-- Column Spacer -->
         <v-col cols="3" class="mb-n6 pa-0">
           <v-fade-transition>
-            <span v-show="showRemovedAllOwnersMsg" class="error-text fs-12 ml-5">Nothing to delete</span>
+            <span v-if="showRemovedAllOwnersMsg" class="error-text fs-12 ml-5">Nothing to delete</span>
```

Comment:
@cameron-eyds replaced **v-show** with **v-if**. To remove elements vs hiding with display: none.